### ISSUE TITLE
Radio Button is Populating Random Text Boxes

### ISSUE THREAD

**Issue by kylebones** [elementor/elementor#7711]
I created two radio button fields in a form, but the second radio field is populating two text boxes in place of the radio buttons.  See form at the bottom of this page: https://crestwoodpools.com/onground-inground-and-above-ground-pools-which-is-best-for-you/

## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Thanks for reporting,

![image](https://user-images.githubusercontent.com/45009641/55459023-2e163080-55f7-11e9-95d8-56a28af979a1.png)


Don't really understand your issue.

It seems OK. 

Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information: 

* Prerequisites.
* **Steps to replicate.**
* **Isolating the problem.**
* **System info.**
* One issue per thread

Without it we simply can't help.

Thanks.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.5.3

Feel free to update!
